compliance_checker/runner.py

```diff
@@ -47,13 +50,25 @@ def run_checker(cls, ds_loc, checker_names, verbose, criteria,
         @returns                If the tests failed (based on the criteria)
         """
         cs = CheckSuite()
-        ds = cs.load_dataset(ds_loc)
+        # using OrderedDict is important here to preserve the order
+        # of multiple datasets which may be passed in
+        score_dict = OrderedDict()
+        if not isinstance(ds_loc, six.string_types):
+            locs = ds_loc
+        # if single dataset, put in list
+        else:
+            locs = [ds_loc]
```

Comment:
ok so this keeps backwards compatibility by not requiring a list to be passed in